# This regexp used to produce a warning. From Perl 5.22, it is an error. Fix the regexp.
# see https://debbugs.gnu.org/cgi/bugreport.cgi?bug=21001
diff --git a/automake.in b/automake.in
index e934c5f..0581aa0 100755
--- a/automake.in
+++ b/automake.in
@@ -4110,7 +4110,7 @@ sub substitute_ac_subst_variables_worker($)
 sub substitute_ac_subst_variables ($)
 {
   my ($text) = @_;
-  $text =~ s/\${([^ \t=:+{}]+)}/&substitute_ac_subst_variables_worker ($1)/ge;
+  $text =~ s/\$\{([^ \t=:+{}]+)}/&substitute_ac_subst_variables_worker ($1)/ge;
   return $text;
 }
 
